Modules/Devices/test/Devices.API.Tests.Integration/API/IdentitiesApi.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net;
+using Devices.API.Tests.Integration.Models;
+using Microsoft.AspNetCore.Http;
+using RestSharp;
+
+namespace Challenges.API.Tests.Integration.API;
```

Comment:
Wrong namespace. There are some other files in this project which have the wrong namespace as well. Please check all of them.